test_msgs/src/test_msgs/service_fixtures.py

```diff
@@ -97,11 +100,58 @@ def get_msg_basic_types():
     return srvs
 
 
+def get_msg_arrays():
+    basic_types_msgs = get_messages_basic_types()
+    srvs = []
+    req = Arrays.Request()
+    req.bool_values = [False, True, False]
+    req.char_values = [0, 255, 0]
+    req.byte_values = [bytes([0]), bytes([255]), bytes([0])]
+    req.float32_values = [0.0, 1.125, -2.125]
+    req.float64_values = [0.0, 1.125, -2.125]
+    req.int8_values = [0, 127, -128]
+    req.uint8_values = [0, 255, 0]
+    req.int16_values = [0, 32767, -32768]
+    req.uint16_values = [0, 65535, 0]
+    req.int32_values = [0, 2147483647, -2147483648]
+    req.uint32_values = [0, 4294967295, 0]
+    req.int64_values = [0, 9223372036854775807, -9223372036854775808]
+    req.uint64_values = [0, 18446744073709551615, 0]
+    req.string_values = ['', 'max value', 'min value']
+    req.basic_types_values[0] = basic_types_msgs[0]
+    req.basic_types_values[1] = basic_types_msgs[1]
+    req.basic_types_values[2] = basic_types_msgs[2]
+
+    resp = Arrays.Response()
+    resp.bool_values = [True, False, False]
+    resp.char_values = [255, 0, 0]
+    resp.byte_values = [bytes([255]), bytes([0]), bytes([0])]
+    resp.float32_values = [1.125, 0.0, -2.125]
+    resp.float64_values = [1.125, 0.0, -2.125]
+    resp.int8_values = [127, 0, -128]
+    resp.uint8_values = [255, 0, 0]
+    resp.int16_values = [32767, 0, -32768]
+    resp.uint16_values = [65535, 0, 0]
+    resp.int32_values = [2147483647, 0, -2147483648]
+    resp.uint32_values = [4294967295, 0, 0]
+    resp.int64_values = [9223372036854775807, 0, -9223372036854775808]
+    resp.uint64_values = [18446744073709551615, 0, 0]
+    resp.string_values = ['max value', '', 'min value']
+    resp.basic_types_values[0] = basic_types_msgs[1]
+    resp.basic_types_values[1] = basic_types_msgs[0]
+    resp.basic_types_values[2] = basic_types_msgs[2]
+
+    srvs.append([req, resp])
+    return srvs
+
+
 def get_test_srv(service_name):
     if 'Empty' == service_name:
         srv = get_msg_empty()
     elif 'BasicTypes' == service_name:
         srv = get_msg_basic_types()
+    elif 'Arrays' == service_name:
+        srv = get_msg_arrays()
```

Comment:
Nitpick: alphabetical order might make more sense here (would also swap the two existing ones).

Reply:
2f0076c